docs/fundamentals/code-analysis/quality-rules/ca1416.md

```diff
@@ -41,6 +41,13 @@ This rule is enabled by default only for projects targeting .NET 5.0 or later. H
 
 Accessing an APIs annotated with the above attributes from the context of a different platform could cause violations.
 
+### TFM target plaforms
+
+The analyzer does not check TFM target platforms from MSBuild properties such as <`TargetFramework>` or `<TargetFrameworks>`, if the TFM has a target platform MSBuild would inject `SupportedOSPlatform` attribute with the targeted platform name in the `AssemblyInfo.cs` file which is consumed by the analyzer. For example if the TFM is `net5.0-windows`, MSBuild would inject `[assembly: System.Runtime.Versioning.SupportedOSPlatform("windows7.0")]` attribute into the `AssemblyInfo.cs` and the entire assembly would considered to be windows only. Therefore calling a windows only APIs versioned with 7.0 or below from windows targeted project would not cause any warning.
+
+  > [!NOTE]
+  > In case the `AssemblyInfo.cs` file generation is disabled for the project (i.e. `<GenerateAssemblyInfo>false</GenerateAssemblyInfo>`) the required assembly level `SupportedOSPlatform` attribute could not be added by MSBuild, therefore you could see warnings for a platform specific APIs usage even if you are targeting that platform. In this case you might want to enable the `AssemblyInfo.cs` file generation or add the attribute manually in your project.
```

Comment:
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/i/ie

```suggestion
  > In case the *AssemblyInfo.cs* file generation is disabled for the project (that is, `<GenerateAssemblyInfo>false</GenerateAssemblyInfo>`), the required assembly level `SupportedOSPlatform` attribute could not be added by MSBuild, therefore you could see warnings for a platform specific APIs usage even if you are targeting that platform. In this case, you might want to enable the *AssemblyInfo.cs* file generation or add the attribute manually in your project.
```